model_api/python/openvino/model_api/models/classification.py

```diff
@@ -59,6 +75,7 @@ def __init__(self, inference_adapter, configuration=None, preload=False):
         self.out_layer_names = ["indices", "scores"]
         if self.output_raw_scores:
             self.out_layer_names.append("raw_scores")
+        _append_xai_names(self.outputs.keys(), self.out_layer_names)
```

Comment:
Will additional output survive `addOrFindSoftmaxAndTopkOutputs` call?

Reply:
I mean at the first glance addOrFindSoftmaxAndTopkOutputs removes extra outputs from the ov model, like saliency map

Reply:
There's 
```python
    for output in inference_adapter.model.outputs:
        if (
            _saliency_map_name in output.get_names()
            or _feature_vector_name in output.get_names()
        ):
            results_descr.append(output)
```
to prevent this